doc/development_process/release_process.rst

```diff
@@ -263,8 +265,7 @@ steps:
         #. Tag and push the version, using an annotated tag::
 
             $ git pull
-            $ git tag -a v1.11.0-rc1
-            <Use "Zephyr 1.11.0-rc1" as the tag annotation>
+            $ git tag -m "Zephyr 1.11.0-rc1" -s v1.11.0-rc1
```

Comment:
Why is this not -a? I thought we wanted annotated tags?

Reply:
From the git tag manual page:
```
Tag objects (created with -a, -s, or -u) are called "annotated" tags
```

So -s will create an annotated tag.

Reply:
Right, good to know!